### PR TITLE
[RDKDEV-309]: DTV: sync-up changes from metro version

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: DTV/DTV.h [hunk 1]
```diff
@@ -122,9 +109,7 @@ namespace WPEFramework
                      Add(_T("serviceList"), &ServiceList);
                   }
 
-                  ~Data()
-                  {
-                  }
+                  ~Data() override = default
```
Comment (steveford-obs) [inline DTV/DTV.h:112]: Semicolon is missing at end of line

Comment (HaseenaSainul) [reply DTV/DTV.h]: done

File: DTV/DTV.h [hunk 2]
```diff
@@ -44,35 +44,29 @@ namespace WPEFramework
...
                   BEGIN_INTERFACE_MAP (Notification)
                      INTERFACE_ENTRY (RPC::IRemoteConnection::INotification)
                   END_INTERFACE_MAP
 
-                  virtual void Activated(RPC::IRemoteConnection*)
+                  void Activated(RPC::IRemoteConnection*) override
                   {
                   }
-                  virtual void Deactivated(RPC::IRemoteConnection *connection)
+                  void Deactivated(RPC::IRemoteConnection *connection) overrride
```
[the remark this answers is not in the document]
Comment (HaseenaSainul) [reply DTV/DTV.h]: done

**Comment by pradeeptakdas** [comment]
Internal JIRA for tracking https://ccp.sys.comcast.net/browse/RDKCOM-3108

**Comment by anand-ky** [comment]
@steveford-obs - Checking if the changes are good to be merged.

**Review by steveford-obs — APPROVED** [review]

**Review by steveford-obs — APPROVED** [review]

**Review by steveford-obs — APPROVED** [review]

**Pull request merged**